chart/values.yaml

```diff
@@ -2,13 +2,13 @@
 # Licensed under the MIT License. See License-MIT.txt in the project root for license information.
 
 # version of Gitpod will be set by the CI job on publishing to https://charts.gitpod.io/
-version:
+version: 0.8.1
```

Comment:
Leave it blank, prod or stage will override this

Reply:
this value will be updated by CI/CD